core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java

```diff
@@ -19,7 +19,7 @@ public void whenDeclaringTooLongString_thenCompilationError() {
     
     @Test
     public void whenStoringInFileTooLongString_thenNoCompilationError() throws IOException {

```

Comment:
Please revert this and just create a stringtoolong.txt in src/test/resources

Reply:
sure